app/i18n/en/conf.php

```diff
@@ -97,6 +97,7 @@
 	),
 	'reading' => array(
 		'after_onread' => 'After “mark all as read”,',
+		'always_show_favorites' => 'Show by default all articles in favorites',
```

Comment:
Sorry but no. That's ungrammatical.

> Having "by default" at the end sounded a bit like the "favorites" were "by default" and it didn't make any sense.

Not really. "To X (verb) blah blah blah by default" is regular, idiomatic English. The "by default" is about the verb (to X), not about the "blah blah blah." (But I don't think that's any different in French?)

Anyway, you could always phrase it like this. The "articles" business is to remain consistent with the other settings, not by necessity.

```suggestion
		'always_show_favorites' => 'Show all favorites by default',
```
```suggestion
		'always_show_favorites' => 'Show read and unread favorites by default',
```

Reply:
Ha ha, I was certain to make a mistake with this last-minute change :D

> Not really. "To X (verb) blah blah blah by default" is regular, idiomatic English. The "by default" is about the verb (to X), not about the "blah blah blah." (But I don't think that's any different in French?)

I think French is a bit more permissive here. First form is right, but can be understood in different ways. That's why I moved the "par défaut" after "Afficher", which is also right in French and avoid the confusion. I changed the English version after that.

I'll pick your first suggestion :)

Reply:
Edit: finally not, "Show all favorites by default" makes it sounds like the first "Always show favorites". "Show all articles in favorites" was pointing the location where the option has an impact. I'll stay with my first suggestion and just adapt for French ;)

Reply:
You're right, the original suggestion was best. :+1: